Reference.md

```diff
@@ -391,6 +393,140 @@ callback("Error message");
+| "$ios_url" | The replacement URL for the App Store to send the user if they don't have the app. _Only necessary if you want a mobile web splash_
+| "$ipad_url" | Same as above but for iPad Store
+| "$fire_url" | Same as above but for Amazon Fire Store
+| "$blackberry_url" | Same as above but for Blackberry Store
+| "$windows_phone_url" | Same as above but for Windows Store
+| "$after_click_url" | When a user returns to the browser after going to the app, take them to this URL. _iOS only; Android coming soon_
+
+You have the ability to control the direct deep linking of each link as well:
+
+| Key | Value
+| --- | ---
+| "$deeplink_path" | The value of the deep link path that you'd like us to append to your URI. For example, you could specify "$deeplink_path": "radio/station/456" and we'll open the app with the URI "yourapp://radio/station/456?link_click_id=branch-identifier". This is primarily for supporting legacy deep linking infrastructure.
+
+#### Usage
+```js
+branch.deepview(
+    data,
+    options,
+    callback (err)
+);
+```
+
+#### Example
+```js
+branch.deepview(
+    {
+        channel: 'facebook',
+        data: {
+            mydata: 'content of my data',
+            foo: 'bar',
+            '$desktop_url': 'https://en.wikipedia.org/wiki/Internet',
+            '$ios_url': 'https://en.wikipedia.org/wiki/Internet',
+            '$ipad_url': 'https://en.wikipedia.org/wiki/Internet',
+            '$android_url': 'https://en.wikipedia.org/wiki/Internet',
+            '$deepview_path': 'item_id=12345',
+            '$og_app_id': '12345',
+            '$og_title': 'My App',
+            '$og_description': 'My app\'s description.'
+        },
+        feature: 'dashboard',
+        stage: 'new user',
+        tags: [ 'tag1', 'tag2' ],
+    },
+    {
+        make_new_link: true,
+        open_app: true
+    },
+    function(err) {
+        console.log(err || 'no error');
+    }
+);
+```
+
+##### Callback Format
+```js
+callback(
+    "Error message"
+);
+```
+
+
+
+### deepviewCta() 
+
+Perform the branch deepview CTA (call to action) on mobile. Namely, depends on how
+*branch.deepview* is set up, the mobile users are redirected accordingly. If the deepview is
+configured with the option *`open_app`* being true, an immediate attempt is made as soon as
+deepview finishes, and thus the CTA is to visit the platform-appropriate app stores; if on the
+other hand, the deepview is configured with the option *`open_app`* being false, the CTA is to
+try to open app, and to visit the platform-appropriate app stores if the open-app attempt failes.
+
+If *branch.deepview* has not been called, an error will arise with a reminder to call
+*branch.deepview* first.
+
+##### Usage
+```js
+$('a.deepview-cta').click(branch.deepviewCta); // If you are using jQuery
+
+document.getElementById('my-elem').onClick = branch.deepviewCta; // Or generally
+
+<a href='...' onclick='branch.deepviewCta()'> // In HTML
```

Comment:
We probs don't want them to have an `href=` if they are overriding onclick with Branch stuff.